mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -1676,6 +1710,77 @@ resource "google_compute_subnetwork" "test" {
 `, envName, network, subnetwork)
 }
 
+func testAccComposerEnvironment_composerV2HighResilience(envName, network, subnetwork string) string {
+	return fmt.Sprintf(`
+resource "google_composer_environment" "test" {
+  name   = "%s"
+  region = "us-east1"
+
+    config {
+      node_config {
+        network          = google_compute_network.test.self_link
+        subnetwork       = google_compute_subnetwork.test.self_link
+      }
+
+      software_config {
+        image_version = "composer-2-airflow-2"
+      }
+
+      workloads_config {
+        scheduler {
+          cpu         = 1.25
+          memory_gb   = 2.5
+          storage_gb   = 5.4
+          count       = 2
+        }
+        web_server {
+          cpu         = 1.75
+          memory_gb   = 3.0
+          storage_gb   = 4.4
+        }
+        worker {
+          cpu         = 0.5
+          memory_gb   = 2.0
+          storage_gb   = 3.4
+          min_count   = 2
+          max_count   = 5
+        }
+				<% unless version == "ga" -%>
+				triggerer {
+					cpu         = 0.5
+					memory_gb   = 2.0
+					count   		= 2
+				}
+				<% end -%>
+      }
+      environment_size = "ENVIRONMENT_SIZE_MEDIUM"
+      resilience_mode = "HIGH_RESILIENCE"
+	  private_environment_config {
```

Comment:
is private config needed for high resilience?

Reply:
yes, HR requires private config.